rest_framework/__init__.py

```diff
@@ -29,13 +29,5 @@
     default_app_config = 'rest_framework.apps.RestFrameworkConfig'
 
 
-class RemovedInDRF313Warning(DeprecationWarning):
-    pass
-
-
-class RemovedInDRF314Warning(PendingDeprecationWarning):
-    pass
-
-
 class RemovedInDRF315Warning(PendingDeprecationWarning):
```

Comment:
Worth noting that `PendingDeprecationWarning` is probably the wrong parent class, but I made a separate PR for that: https://github.com/encode/django-rest-framework/pull/8665